modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSet.macro

```diff
@@ -1,5 +1,58 @@
 definition {
 
+	macro chooseOptionNewFilterDropdown {
+		var key_option = ${key_option};
+
+		Type(
+			locator1 = "FrontendDataSetAdmin#NEW_FILTER_SELECT_OPTION",
+			value1 = ${key_option});
+	}
```

Comment:
Is it possible to use DropdownMenuItem.click(menuItem = "ItemName"); instead of creating a new macro?